### PR TITLE
 Convert hui-image-element to TypeScript/LitElement

### PR DESCRIPTION
**Pull request by iantrich**
Dependent on https://github.com/home-assistant/home-assistant-polymer/pull/1871

Will rebase after 1871 merged

### CODE REVIEW

File: src/panels/lovelace/elements/hui-image-element.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        .entity="${this._config.entity}"
+        .image="${this._config.image}"
+        .stateImage=${this._config.state_image}
+        .cameraImage=${this._config.camera_image}
+        .filter=${this._config.filter}
+        .stateFilter=${this._config.state_filter}
+        .title="${computeTooltip(this.hass!, this._config)}"
+        .aspectRatio="${this._config.aspect_ratio}"
+        @ha-click="${() => handleClick(this, this.hass!, this._config!, false)}"
+        @ha-hold="${() => handleClick(this, this.hass!, this._config!, true)}"
```
Comment (zsarnett) [inline src/panels/lovelace/elements/hui-image-element.ts]: Again do you need the `!`? You may need them, more a question

Comment (iantrich) [reply src/panels/lovelace/elements/hui-image-element.ts]: I do on hass, not on _config (you would think) but linter complains when I don't. 🤷‍♂️ (BTW shrug is now my go to emoji)

**Comment by iantrich** [comment]
Rebase complete

File: src/panels/lovelace/elements/hui-image-element.ts (hunk 1 above)
Comment (zsarnett) [reply src/panels/lovelace/elements/hui-image-element.ts]: Check for `!hass` with the config check above

Comment (iantrich) [reply src/panels/lovelace/elements/hui-image-element.ts]: @balloob can you revisit this and make sure I'm remembering correctly?

**Review by balloob — APPROVED** [review]

**Pull request merged**